srcs/back/src/friends/friends.controler.ts

```diff
@@ -0,0 +1,33 @@
+import { Body, Controller, Get, Post, UseGuards } from "@nestjs/common";
+import { AuthGuard } from "@nestjs/passport";
+import { FriendsService } from "./friends.service";
+import { JwtPayload } from "src/auth/jwt/jwt-auth.strategy";
+import { UserDecorator } from "src/users/decorator/users.decorator";
+
+@UseGuards(AuthGuard)
+@Controller('friends')
+export class FriendsControler {
+  constructor(
+    private friendsService: FriendsService
+  ) { }
+
+  @Post('new')
+  addNewFriend(@UserDecorator() user: JwtPayload, @Body() payload: { userId: number }) {
```

Comment:
```suggestion
  addNewFriend(@UserDecorator() user: JwtPayload, @Body() payload: { userId: number }) {
```
il faut valider le body en faisant un DTO (il y a quelques examples dans le code)